libs/ledger-live-common/src/wallet-api/Exchange/server.ts

```diff
@@ -192,8 +261,8 @@ export const handlers = ({
               exchangeType: ExchangeType[params.exchangeType],
               provider: params.provider,
               transaction: tx,
-              signature: Buffer.from(params.hexSignature, "hex").toString(),
-              binaryPayload: Buffer.from(params.hexBinaryPayload, "hex").toString(),
+              signature: params.hexSignature,
+              binaryPayload: params.hexBinaryPayload,
```

Comment:
Isn't this change a bit breaking ?
And from the variable names it looks correct to go from the hex string to a binary or utf8 string

Reply:
Ok yeah a rename might good